gametypes/ctftactics/ctf_players.as

```diff
@@ -527,6 +528,8 @@ class cPlayer
                 this.client.selectWeapon( WEAP_NONE );
                 this.client.pmoveFeatures = this.client.pmoveFeatures & ~PMFEAT_WEAPONSWITCH;
                 this.ent.effects |= EF_PLAYER_HIDENAME;
+				// The player should be visible for bots only on range of 192 units
+				this.ent.aiVisibilityDistance = 192.0f;
```

Comment:
I will fix it but tbh the original files were messy at it too... All scripts need a global style fix.